src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentService.kt

```diff
@@ -47,24 +47,19 @@ class CodyAgentService(project: Project) : Disposable {
         FixupService.getInstance(project).getSessionForTask(task)?.update(task)
       }
 
-      agent.client.onEditTaskDidDelete = Consumer { task ->
```

Comment:
Looks like you reverted my change to the callbacks/lifecycle managenet.
Was that intentional?
Initialising it in `FixupService` won't work in some cases, e.g. after agent restart.
If that was not intentional, please revert those changes.

Reply:
@steveyegge pinging this comment ^